app/assets/javascripts/modules/namespaces/services/namespaces.js

```diff
@@ -36,9 +36,9 @@ function searchTeam(teamName) {
   return resource.teamTypeahead({ teamName });
 }
 
-function existsByName(name) {
-  return resource.existsByName({ name })
-    .then(() => true)
+function validate(name) {
+  return resource.validate({ name })
+    .then((response) => { return response.data })
     .catch((response) => {
       if (response.status === 404) {
```

Comment:
Is 404 happening somehow in this endpoint? I don't think so. We can remove this and handle the possible responses properly.